dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/hive/HiveMetadataConnector.java

```diff
@@ -384,8 +385,11 @@ public void addTasksTo(List<? super Task<?>> out, @Nonnull ConnectorArguments ar
     out.add(new SchemataTask(databasePredicate));
     out.add(new TablesJsonTask(databasePredicate, shouldDumpPartitions));
     out.add(new FunctionsTask(databasePredicate));
-    out.add(new CatalogsTask());
-    out.add(new DatabasesJsonlTask());
+    if (BooleanUtils.toBoolean(
```

Comment:
What is a reason for omitting metadata extraction by default? Does it cause any issues or is time consuming?

This change is not backward compatible because it changes the default tool behavior. I'd recommend avoiding such changes to the existing connectors, but this may be fine if we are sure this is not used by the customers.

Reply:
The extraction may take a long time, especially for tables and partitions it doesn't make sense to extract them twice. This is not used by customers yet, so for now I'm introducing a boolean flag for testing purposes. Later we can change this.